tests/behavior_test.cpp

```diff
@@ -183,18 +185,28 @@ TEST_CASE( "check_npc_behavior_tree", "[npc][behavior][!mayfail]" )
 
 TEST_CASE( "check_monster_behavior_tree", "[monster][behavior]" )
 {
+    const tripoint monster_location( 5, 5, 0 );
+    clear_map();
+    monster &test_monster = spawn_test_monster( "mon_locust", monster_location );
+
+    behavior::monster_oracle_t oracle( &test_monster );
     behavior::tree monster_goals;
-    monster_goals.add( &string_id<behavior::node_t>( "monster_special" ).obj() );
-    monster &test_monster = spawn_test_monster( "mon_zombie", { 5, 5, 0 } );
+    monster_goals.add( test_monster.type->get_goals() );
+
     for( const std::string &special_name : test_monster.type->special_attacks_names ) {
         test_monster.reset_special( special_name );
     }
-    behavior::monster_oracle_t oracle( &test_monster );
     CHECK( monster_goals.tick( &oracle ) == "idle" );
+    for( const tripoint &near : g->m.points_in_radius( monster_location, 1 ) ) {
```

Comment:
It is ridiculous, but [you can't use `near` as variable name in VS](https://stackoverflow.com/questions/8948493/visual-studio-doesnt-allow-me-to-use-certain-variable-names).

![image](https://user-images.githubusercontent.com/16213433/82801738-74c38b80-9e86-11ea-8c41-64051df7bf10.png)

Reply:
Gah, yes I've seen that before, I need to add a lgtm filter or something to warn on this and other VS reserved words.